ui-v2/app/styles/components/composite-row/layout.scss

```diff
@@ -50,15 +50,12 @@
   display: inline-flex;
   flex-wrap: nowrap;
 }
-%composite-row-detail dt {
```

Comment:
Not sure if there was a reasoning for this, but I need to display `dt` to show the icons that also hold the Tooltip.

Reply:
I think in the previous iteration where I'd started adding `dt`s we were actually showing the content of the dt - now that we are this make perfect sense to remove 👍